electron/electron/main.ts

```diff
@@ -0,0 +1,48 @@
+import { app, BrowserWindow } from 'electron'
+import * as path from 'path'
+import * as url from 'url'
+import installExtension, { REACT_DEVELOPER_TOOLS, REDUX_DEVTOOLS } from 'electron-devtools-installer'
+
+let mainWindow: Electron.BrowserWindow | null
+
+function createWindow () {
+  mainWindow = new BrowserWindow({
+    width: 1100,
+    height: 700,
```

Comment:
A future feature could be to make this relative to the device resolution and to add support for different aspect ratios or orientations.